pkgs/tools/text/ispell/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchurl, bison, ncurses }:
+
+stdenv.mkDerivation rec {
+  name = "ispell-3.3.02";
```

Comment:
This is outdated, current version is 3.4.00: https://www.cs.hmc.edu/~geoff/ispell.html

Reply:
Yes.  The reason why I did not try to use the newest one is that ispell here is only used as a build dependency for the dictionary.  So I basically stuck to the version that someone else deemed working.